components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java

```diff
@@ -5058,6 +5058,16 @@ public static boolean isTenantAvailable(String tenantDomain) throws UserStoreExc
                 .isTenantActive(tenantId);
     }
 
+    public static boolean isOrganizationTenantAvailable(String organization) throws UserStoreException {
```

Comment:
this method won't be needed

Reply:
Done via https://github.com/wso2/carbon-apimgt/pull/10539/commits/d9368dcf8df1c26ded2c9ff81b077a5267d30044